St Gabriel School material

JRigelo opened the pull request:
let's try again...

Commit:
Hello World class with python session

Commit:
change to my repo the binder address

Commit:
material for day one

Commit:
change image path

Commit:
add data to this folder

Commit:
create 1 hr version for elementary grades

Commit:
icecream pic

Commit:
add ice cream notebook

Commit:
ice cream dataset

Commit:
correct notebook

Commit:
delete extra files

Commit:
update readme

Commit:
add Clay's binder path back to pull request

Commit:
add back for pull request

Commit:
Charles' updated version

SirDudeness commented:
I think some changes that Charlie made on his Elementary file is conflicting with yours. Could you rename your elementary file different from his so it maybe treats it as a new file instead?

JRigelo commented:
I will try that

On Tue, Oct 18, 2016 at 9:43 PM, Clay Porter notifications@github.com
wrote:

> I think some changes that Charlie made on his Elementary file is
> conflicting with yours. Could you rename your elementary file different
> from his so it maybe treats it as a new file instead?
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> https://github.com/SirDudeness/HelloWorld/pull/3#issuecomment-254695653,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AILJalJsH0mqTzoFlgSgl2OPq3nQtYGtks5q1YPugaJpZM4Kaf99
> .

JRigelo commented:
Didn't work...I think the problem is that at some point I deleted Charle's
file, so now even though I added the new version manually it is not
enough...

On Tue, Oct 18, 2016 at 9:44 PM, Joyce Rigelo jrigelo@utexas.edu wrote:

> I will try that
> 
> On Tue, Oct 18, 2016 at 9:43 PM, Clay Porter notifications@github.com
> wrote:
> 
> > I think some changes that Charlie made on his Elementary file is
> > conflicting with yours. Could you rename your elementary file different
> > from his so it maybe treats it as a new file instead?
> > 
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub
> > https://github.com/SirDudeness/HelloWorld/pull/3#issuecomment-254695653,
> > or mute the thread
> > https://github.com/notifications/unsubscribe-auth/AILJalJsH0mqTzoFlgSgl2OPq3nQtYGtks5q1YPugaJpZM4Kaf99
> > .
